docs/feature_combo.md

```diff
@@ -184,18 +173,18 @@ bool get_combo_must_hold(uint16_t index, combo_t *combo) {
 }
 ```
 
-#### Variable Length Combos
-If `COMBO_VARIABLE_LEN` is defined in `config.h`, it allows the user to programmatically declare the size of the Combo data structure and avoid updating COMBO_COUNT. Instead a variable is exported called COMBO_LEN and is usually set with something similar to the following in `keymap.c`: `int COMBO_LEN = sizeof(key_combos) / sizeof(key_combos[0]);` or by adding `COMBO_LENGTH` as the *last* entry in the combo enum and then `int COMBO_LEN = COMBO_LENGTH;` as such:
+## Variable Length Combos
+If you leave `COMBO_COUNT` undefined in `config.h`, it allows the you to programmatically declare the size of the Combo data structure and avoid updating COMBO_COUNT. Instead a variable called `COMBO_LEN` has to be set. It can be set with something similar to the following in `keymap.c`: `uint16_t COMBO_LEN = sizeof(key_combos) / sizeof(key_combos[0]);` or by adding `COMBO_LENGTH` as the *last* entry in the combo enum and then `uint16_t COMBO_LEN = COMBO_LENGTH;` as such:
```

Comment:
> it allows the you to programmatically

either stick with the old "the user" or remove the superfluous "the" in front of "you"